src/webgpu/shader/validation/wgsl/variable-with-initilizer-storage.fail.wgsl

```diff
@@ -0,0 +1,13 @@
+# v-0032: variable 'u' has an initializer, however its storage class is 'storage'.
+
+[[block]]
+struct PositionBuffer {
+  [[offset(0)]] pos: vec2<f32>; 
+};
+
+[[group(0), binding(0)]]
```

Comment:
I'm probably missing something, tint ouput:
variable-with-initilizer-storage.fail.wgsl:8:3 error: expected decoration
[[group(0), binding(0)]]
  ^^^^^

Reply:
This is tint falling behind the spec, tint is still looking for `set` not `group`. So, this is correct and Tint needs to be updated.